src/core/client/admin/components/StoryInfoDrawer/StorySettingsContainer.tsx

```diff
@@ -90,13 +88,34 @@ const StorySettingsContainer: FunctionComponent<Props> = ({
               handleSubmit,
               submitting,
               dirty,
-              form,
               submitSucceeded,
               dirtySinceLastSubmit,
             }) => {
               return (
                 <form onSubmit={handleSubmit}>
                   <Flex direction="column">
+                    <Flex direction="row" className={styles.setting}>
+                      {/* PREMOD ALL COMMENTS ENABLED */}
+                      <Field
+                        name="premoderateComments"
```

Comment:
Can we make the field name here `moderation` and then use a `parse` and `format` like in `PremodConfig.tsx`, for example, to change to/from a boolean? This would be consistent with how we're handling this in other places, and I think also more readable.

Reply:
agreed!